### DISCUSSION TITLE
Disable comment detection (#)

### DISCUSSION THREAD

**Discussion by g-hauer** [MarkPflug/Sylvan#116] [Q&A] [answered]
Is there any chance to disable comment detection/interpretation when reading data ('#')?

it seems only possible to change the comment character to another, but not to none...

**Comment by MarkPflug** [comment, marked as answer]
You can set it to the null character `'\0'`, which should work as long as your data doesn't have one in it, which would be very strange indeed.

**Reply by g-hauer** [reply]
thank u very much for confirming my suspicion
